tests/client/coll.rs

```diff
@@ -14,7 +12,7 @@ fn find_and_insert() {
 
     // Insert document
     let doc = doc! {
-        "title" => Bson::String("Jaws".to_owned())
+        "title" => (Bson::String("Jaws".to_owned()))
```

Comment:
Actually, from re-examining it, this might be trickier to change than I thought. I'll have to spend some time to see if getting rid of the parentheses is possible.